app/routes/index.md

```diff
@@ -3,4 +3,47 @@ meta:
   title: GCN - General Coordinates Network
 ---
 
-# Welcome to the General Coordinates Network
+# Welcome to the General Coordinates Network (GCN)
+
+<img src="https://gcn.gsfc.nasa.gov/GCN.gif" width="400" align="right" alt="GCN Diagram"/>
+
+GCN is the NASA collaboration platform for transient alerts and community reports from space- and ground-based missions, observatories, and experiments used by the multimessenger and time-domain astrophysics communities. GCN is the primary distributer of detections and follow-up observations of Gamma-ray Bursts (GRBs), gravitational wave compact binary mergers, and high-energy neutrinos.
+
+The new GCN is the next generation transient alert system built upon the legacy of the Gamma-ray Coordinates Network (referred to throughout as GCN Classic, formerly BACODINE/TAN), updated to use the modern technologies for transient distribution being utilized by the optical transient community and providing a robust reliable cloud-based system.
+
+## The GCN system distributes:
+
+- **Notices:** Locations of GRBs and other Transients (e.g. gravitational waves, neutrinos, AGN flares) detected by space- and ground-based observatories (most in realtime while the source is still bursting and others are that delayed due to communications and processing).
+- **Circulars:** Reports of follow-up observations made by ground-based and space-based optical, radio, X-ray, gamma-ray, TeV, and other particle observers.
+
+These two functions provide a one-stop shopping network for follow-up sites and GRB and transient researchers. The new GCN system uses the [Apache Kafka](https://kafka.apache.org) protocol for distributing Notices, and is operated within the Commercial managed Kafka system, [Confluent Platform](https://www.confluent.io), running on Amazon Web Services. The Kafka systems utilizes Hopclient, developed by SCiMMA, and ...
```

Comment:
🤣 "shopping network"? Are we producing infomercials?

Reply:
I copied this from the old GCN page.  I'll change it.